packages/react/src/2016-12-01/select.js

```diff
@@ -0,0 +1,69 @@
+
+/**
+Copyright 2016 Autodesk,Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+require('@orion-ui/components/lib/2016-12-01/select');
+const { Skins } = require('@orion-ui/style/lib/2016-12-01');
+
+const React = require('react');
+const applyProps = require('@orion-ui/components/lib/utils/apply-props');
+
+const PropTypes = React.PropTypes;
+
+class Select extends React.Component {
+  constructor(props) {
+    super(props);
+    this.updateState = this.updateState.bind(this);
+    this.registerListeners = this.registerListeners.bind(this);
+  }
+
+  componentDidMount() {
+    applyProps(this._el, this.props);
+  }
+
+  componentWillReceiveProps(props) {
+    applyProps(this._el, props);
+  }
+
+  registerListeners(el) {
+    this._el = el;
+
+    this._el.addEventListener('change', this.updateState);
```

Comment:
Don't register change here, we want the consumer to handle that. See https://github.com/orion-ui/orion/blob/500a3efbcfce0c7b84beb5646446bbfe760f50ab/packages/storybook/stories/select.js#L618 for an example of how it should look to consume this component in React.